app/views/registration_wizard/dont_have_teacher_reference_number.html.erb

```diff
@@ -13,75 +13,91 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">Get a Teacher Reference Number (TRN)</h1>
 
+    <h2 class="govuk-heading-m">Why do I need this?</h2>
+
+    <p class="govuk-body">
+      Everyone needs a <%= govuk_link_to "TRN (opens in a new tab)", "https://www.gov.uk/guidance/teacher-reference-number-trn", target: :_blank, rel: "noopener noreferrer" %>
+      to register for an NPQ. You do not need to be a teacher to get a TRN.
+    </p>
+
     <h2 class="govuk-heading-m">Request a TRN by email</h2>
 
     <p class="govuk-body">
-      If you have never been given a TRN, then email the Teaching Regulation Agency at <%= govuk_mail_to "qts.enquiries@education.gov.uk", "qts.enquiries@education.gov.uk" %>. The team aim to respond to valid email requests within 5 working days
+      If you’ve never been given a TRN, email the Teaching Regulation Agency at <%= govuk_mail_to "qts.enquiries@education.gov.uk", "qts.enquiries@education.gov.uk" %>.
+      The team aim to respond to valid email requests within 5 working days.
     </p>
 
     <p class="govuk-body">
-      You will need to provide contact details and proof of I.D. to receive a TRN.
+      You’ll need to provide contact details and proof of ID to receive a TRN.
     </p>
 
     <p class="govuk-body">
-      Use the free Galaxkey secure email platform to ensure your identification documents are protected.
+      Use the free Galaxkey secure email platform to ensure your identification documents are protected. Full instructions can be found below.
     </p>
 
-    <h2 class="govuk-heading-m">Send a secure email</h2>
+    <h2 class="govuk-heading-m">Register to send a secure email</h2>
 
     <p class="govuk-body">
-      To send a secure email to the department, please go to the Galaxkey site: <%= govuk_link_to "https://manager.galaxkey.com/services/registerme", "https://manager.galaxkey.com/services/registerme" %>
+      To send a secure email to the department, please go to the Galaxkey site:
+      <%= govuk_link_to "https://manager.galaxkey.com/services/registerme", "https://manager.galaxkey.com/services/registerme",  target: :_blank, rel: "noopener noreferrer" %>
+      (opens in a new tab)
```

Comment:
are we able to make this consistent in the PR. I don't know or mind which way round it should be. but either the `(opens in a new tab)` part lives inside or outside of the link

Reply:
i always do within the link. Just though it looked a bit weird in the link with the url but yeah consistency over style in this case.